core/src/commonMain/kotlin/com/sourcepoint/mobile_core/network/SourcepointClient.kt

```diff
@@ -44,6 +46,8 @@ import kotlin.reflect.KSuspendFunction1
 interface SPClient {
     suspend fun getMetaData(campaigns: MetaDataRequest.Campaigns): MetaDataResponse
 
+    suspend fun postPvData(request: PvDataRequest): PvDataResponse
```

Comment:
Can you add `@Throws(Exception::class)` behind the function signature? (I'm adding it to all network calls, except of error metrics).